public/base-locales/pl.json

```diff
@@ -1,50 +1,50 @@
 {
-  "login": "Login",
+  "login": "Zaloguj się",
   "click_once": "Kliknij 'Autoryzuj' tylko raz",
   "save": "Zapisz",
   "reset": "Zresestuj",
   "close": "Zamknij",
-  "reset_filters": "Zresetuj filtry",
+  "reset_filters": "Reset ustawień",
```

Comment:
I would leave it as `Zresetuj filtry`. As it's related only to filters, not settings.

Reply:
so resetting won't change icon styles etc? 🤔